Azure-Samples/ms-identity-android-java
# Algo 1/fix vulnerabilities

## Purpose

Security Update - vulnerability fix by Upgrading MSAL from 4.+ to 5.6.0

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: Security Update - vulnerability fix by Upgrading MSAL from 4.+ to 5.6.0
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout algo-1/fix-vulnerabilities
```

* Test the code


```
- open in android studio / equivalent editor
- follow the README but ensure redirect URI is set in the config files and android manifest. 
- client id is also required. Hence the app registration is also required. The registered app must be multi-tenant.
```

## What to Check
Verify that the sign in, sign out and scoped API calls are working as expected.
